.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+## What are the relevant tasks?
+
+## What does this PR do & what background information is important for this PR?
+
+## Checklist
+- [ ] Updated documentation (if required, see README.md)
+- [ ] Ran locally: `npm run lint && npm run test-coverage`
```

Comment:
nit: if this is a pre-commit hook, is it necessary to include in the checklist? won't be possible to commit if the lint or test cov fails ...

Reply:
Well, the reason I did this is because I believe if you don't run npm install, i.e. just edit the SDK and push it up you wont ever hit these git commit hooks.